Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM nginx:stable-alpine
+
+RUN mkdir -p /var/cache/nginx/client_temp /var/run/nginx
+RUN chown -R nginx:nginx /var/cache/nginx /var/run/nginx /usr/share/nginx/html
+
+COPY ./nginx.conf /etc/nginx/nginx.conf
+COPY ./angular/dist/angular-starter/ /usr/share/nginx/html
+
+USER nginx
+
+CMD ["nginx", "-g", "daemon off;"]
```

Comment:
configuration of running nginx should rely on config file i.e. nginx.conf, not arguments that you pass in CMD. Otherwise you reduce your maintainability. Your nginx config file should be the ONLY source of truth.

The more options you spread in different files - the more complex the maintenance will be in the future and will require more time.

Reply:
Done